dace/sourcemap.py

```diff
@@ -49,12 +50,13 @@ def tmp_location(name: str) -> str:
         :param name: name of the SDFG
         :return: path to the tmp folder
     """
-    return os.path.abspath(os.path.join(".dacecache", name, "map", "tmp.json"))
+    build_folder = Config.get('default_build_folder')
+    return os.path.abspath(os.path.join(build_folder, name, "map", "tmp.json"))
 
 
 def temporaryInfo(name: str, data):
     """ Creates a temporary file that stores the json object
-        in the map folder (.dacecache/{SdfgName}/map)
+        in the map folder ({build_folder}/{sdfg_name}/map)
```

Comment:
Is this some sort of reference or formatting string? If not, I would use `<build folder>/<SDFG name>/map` so it doesn't look like a fstring gone wrong 😛

Reply:
yeah I tried to keep the old docs intact. will fix